windows/installer-source/KaliteSetupScript.iss

```diff
@@ -242,14 +242,42 @@ begin
     end;
 end;
 
-procedure HandleUpgrade(targetPath : String);
+procedure SelectContentFolder;
 var
-    prevVerStr : String;
     userKaliteContent: String;
     systemKaliteDir: String;
     userPath : String;
     contentPath : String;
-    contentDb : String;
+    retCode: Integer;
+begin
+    userPath := ExpandConstant('{%USERPROFILE}')
+    contentPath := ExpandConstant('{%CONTENT_ROOT}')
+    systemKaliteDir := ExpandConstant(userPath + '\.kalite')
+    userKaliteContent := ExpandConstant(systemKaliteDir + '\content');
+    if Not DirExists(userKaliteContent) then
+    begin
+     if Not DirExists(contentPath) then
+        begin
+            if MsgBox('You can also download contents after the installation. If you don`t have a folder with downloaded contents, press Cancel.', mbInformation,  MB_YESNO or MB_DEFBUTTON1) = IDYES then
+            begin 
+               if BrowseForFolder('Please select the .kalite/content/ folder', userPath, False) then

```

Comment:
Is it normal that the default location of `BrowseForFolder` is *not* the same as what's already selected (`userPath` instead of `userKaliteContent`) ?